corehq/apps/app_manager/static/app_manager/js/releases/app_view_release_manager.js

```diff
@@ -77,6 +77,32 @@ hqDefine("app_manager/js/releases/app_view_release_manager", function () {
     if (el.length) {
         el.koApplyBindings(releasesMain);
         _.defer(function () { releasesMain.goToPage(1); });
+
+        var releaseControlEl = $('#release-control');
+        if (releaseControlEl.length) {
+            releasesMain.showReleaseControl(false);
+
+            var setReleaseLockButtons = function () {
+                if (releasesMain.releasesEnabled()) {
+                    $("#btn-release-unlocked").show();
+                    $("#btn-release-locked").hide();
+                } else {
+                    $("#btn-release-unlocked").hide();
+                    $("#btn-release-locked").show();
+                }
+            };
+
+            $("#btn-release-unlocked").click(function () {
+                releasesMain.showReleaseControl(false);
```

Comment:
when we click on unlock, we are passing `false` to `showReleaseControl`. Should not this be opposite?

Reply:
`btn-release-unlocked` is linked to unlock icon and is used to display the current status of the screen i.e. when this button is clicked, releases are already enabled and to be disabled.

Reply:
Right!
Thank you!